include/odp/api/spec/schedule.h

```diff
@@ -157,13 +157,14 @@ int odp_schedule_multi_no_wait(odp_queue_t *from, odp_event_t events[],
 /**
  * Pause scheduling
  *
- * Pause global scheduling for this thread. After this call, all schedule calls
- * will return only locally pre-scheduled events (if any). User can exit the
- * schedule loop only after the schedule function indicates that there's no more
- * (pre-scheduled) events.
- *
- * Must be used with odp_schedule() and odp_schedule_multi() before exiting (or
- * stalling) the schedule loop.
+ * Pause global scheduling for this thread. After this call, only
+ * ODP_SCHED_NO_WAIT schedule calls are allowed and these calls will return only
+ * locally pre-scheduled events (if any). User can exit the schedule loop only
+ * after the schedule function indicates that there's no more (pre-scheduled)
+ * events.
+ *
+ * Must be used with odp_schedule(), odp_schedule_multi(), or
+ * odp_schedule_multi_no_wait() before exiting (or stalling) the schedule loop.
```

Comment:
This refers actually to the main schedule loop (not to sched calls after pause), so the list should cover all odp_schedule calls. It could be better change the text to refer to schedule loop and list e.g. odp_schedule() as an example. Also an example (code) of the required call pattern could be added ( schedule loop -> pause -> schedule no wait until no events -> exit schedule loop).

Reply:
Ok, fixed in V2.